[IMP] sales: updated down payment doc for 16 (#5523)
odoo/documentation: content/applications/sales/sales/invoicing/down_payment.rst

Then, click :guilabel:`Add a product` in the :guilabel:`Order Lines` tab, and select the
:guilabel:`Cabinet with Doors` product.

.. image:: down_payment/cabinet-with-doors-quotation.png
   :align: center
   :alt: Adding cabinet with doors product to a new quotation in the Odoo Sales application.

When the order is confirmed (via the :guilabel:`Confirm` button), the quotation turns into a sales
order. Then, create/view the invoice, by clicking :guilabel:`Create Invoice`.

.. image:: down_payment/cabinet-sales-orders-confirmed.png
   :align: center
   :alt: Cabinet with doors sales order that's been confirmed in the Odoo Sales application.

Next, on the :guilabel:`Create invoices` pop-up window that appears, select :guilabel:`Down payment
(percentage)`, and type `50` in the :guilabel:`Down Payment Amount` field.

.. note::
   The :guilabel:`Income Account` and :guilabel:`Customer Taxes` fields are *not* required fields.

Lastly, click :guilabel:`Create Draft Invoice` to create and view the invoice draft.

.. image:: down_payment/create-invoices-percentage-popup.png
   :align: center
   :alt: Create invoices pop-up window with 50% percentage fields filled in with Odoo Sales.

> Clicking :guilabel:`Create Draft Invoice` reveals the draft invoice, which showcases the down

Reviewer: ```suggestion
Clicking :guilabel:`Create Draft Invoice` reveals the draft invoice, which includes the down
```

Now, when the customer wants to pay the remaining amount of the order, another invoice must be
created. To do that, return to the sales order, via the breadcrumb links.

Back on the sales order, a new :guilabel:`Down Payments` section is present in the :guilabel:`Order
Lines` tab, along with the :guilabel:`Down payment` that was just invoiced and posted.

.. image:: down_payment/down-payments-section-order-lines.png
   :align: center
   :alt: The down payments section in the order lines tab of a sales order.

Next, click the :guilabel:`Create Invoice` button.

On the :guilabel:`Create invoices` pop-up window that appears, there are two new fields:
:guilabel:`Already invoiced` and :guilabel:`Amount to invoice`.

.. image:: down_payment/create-invoices-pop-up-already-invoiced.png
   :align: center
   :alt: The deduct down payment option on the create invoices pop up in Odoo Sales.

If the remaining amount is ready to be paid, the :guilabel:`Regular Invoice` option should be
selected. Odoo will create an invoice the exact amount needed to complete the total payment, as
indicated in the :guilabel:`Amount to invoice` field.

Then, click :guilabel:`Create Draft Invoice`.

> Doing so reveals an another :guilabel:`Customer Invoice Draft` page, listing *all* the invoices for

Reviewer: ```suggestion
Doing so reveals another :guilabel:`Customer Invoice Draft` page, listing *all* the invoices for
```

remaining amount at a later date. In turn, the seller provides goods or services to the buyer after
accepting the down payment, trusting that the remaining amount will be paid later on.

Create invoices
===============

> 

Reviewer 2: Do you think it would add value to this doc if we included brief instructions on creating and confirming a sales order here, before the invoice section? Or is that implied/not necessary in this particular doc?

Author: I understand this suggestion, however, I feel like if a reader is interested in this doc, they will already know how to accomplish those sales order goals. So, I think I'll leave this addition out for now -- but if readers start to reach out, wondering how they can do those things, I'll be sure to add it in, per your request. Thanks for bringing this to my attention. I appreciate it! 👍

When a sales order is confirmed, the option to create an invoice becomes available, via the
:guilabel:`Create Invoice` button, located in the upper-left corner of the sales order form. When
clicked, a :guilabel:`Create invoices` pop-up appears.

.. image:: down_payment/create-invoices-popup-form.png
   :align: center
   :alt: Create invoices pop-up form that appears in Odoo Sales.

.. note::
   Invoices are automatically created as drafts, so they can be reviewed before validation.

On the :guilabel:`Create invoices` pop-up, there are 3 options to choose from in the
:guilabel:`Create Invoice` field:

- :guilabel:`Regular invoice`
- :guilabel:`Down payment (percentage)`

> - :guilabel:`Down payment (fixed amount)`

Reviewer 2: This isn't paramount either, but it might be nice to have a short description/definition of each of these 3 choices

Author: This, too, may be unnecessary. In my opinion, these elements seem fairly self-explanatory. So, for the sake of staying as concise as possible, I'll leave these elements as-is -- but if other reviewers bring this up, I'll make the necessary changes right away. Thanks! 👍

.. note::

>    If :guilabel:`Regular Invoice` is selected the other fields disappear, as they only pertain to

Reviewer 2: ```suggestion
   If :guilabel:`Regular Invoice` is selected, the other fields disappear, as they only pertain to
```

   down payment configurations.

   .. image:: down_payment/create-invoices-popup-form-regular-option.png
      :align: center
      :alt: The Regular Invoice option selected on the Create invoices pop-up form.

Initial down payment request
============================

On the :guilabel:`Create invoices` pop-up form, the down payment options are:

- :guilabel:`Down payment (percentage)`
- :guilabel:`Down payment (fixed amount)`

Once the desired down payment option is selected in the :guilabel:`Create Invoice` field on the

> pop-up form, designate the desired amount (either as a percentage or fixed amount) in the

Reviewer 2: ```suggestion
pop-up form, designate the desired amount, either as a percentage or a fixed amount, in the
```

:guilabel:`Down Payment Amount` field.

Then, select the appropriate income account for the invoice in the :guilabel:`Income Account` field.

> Then, select a tax amount, if necessary, in the :guilabel:`Customer Taxes` drop-down field.

Reviewer 2: ```suggestion
Next, select a tax amount, if necessary, in the :guilabel:`Customer Taxes` drop-down field.
```

.. image:: down_payment/create-invoices-popup-form-filled-out.png
   :align: center
   :alt: A create invoices pop-up form with down payment fields filled in with information.

> When all fields are filled in with the desired information, click the :guilabel:`Create Draft
> Invoice` button. Upon clicking :guilabel:`Create Draft Invoice` on the :guilabel:`Create invoices`
> pop-up form, Odoo reveals the :guilabel:`Customer Invoice Draft`.

Reviewer 2: ```suggestion
Once all fields are filled in with the desired information, click the :guilabel:`Create Draft
Invoice` button. Upon clicking this button, Odoo reveals the :guilabel:`Customer Invoice Draft`.
```
This read as a little long-winded for me, so this is just a suggestion to shorten it a bit

> In the :guilabel:`Invoice Lines` tab of the :guilabel:`Customer Invoice Draft` the down payment that

Reviewer 2: ```suggestion
In the :guilabel:`Invoice Lines` tab of the :guilabel:`Customer Invoice Draft`, the down payment that
```

was just configured in the :guilabel:`Create invoices` pop-up form appears as a :guilabel:`Product`.

.. image:: down_payment/down-payment-product-invoice-draft.png
   :align: center
   :alt: Down payment as a product in the invoice lines tab of a customer invoice draft in Odoo.

.. note::
   When the :guilabel:`Down payment` product in the :guilabel:`Invoice Lines` tab is clicked, Odoo
   reveals the product form for the down payment.

   By default, the :guilabel:`Product Type` of down payment products generated for invoices are set
   as :guilabel:`Service`, with the :guilabel:`Invoicing Policy` set to :guilabel:`Prepaid/Fixed
   Price`.

   .. image:: down_payment/down-payment-product-form.png
      :align: center
      :alt: Down payment product form with service product type and invoicing policy field.

   This product can be edited/modified at any time.

.. warning::
   If :guilabel:`Based on Delivered Quantity (Manual)` is chosen as the :guilabel:`Invoicing
   Policy`, an invoice will **not** be able to be created.

> Request 50% down payment (example)

Reviewer 2: ```suggestion
Example: request 50% down payment
```
Just a suggestion to bring the word "example" to the front. Feel free to ignore if you like it as is

==================================

.. note::
   The following example involves a 50% amount down payment on a product (:guilabel:`Cabinet with
   Doors`) with :guilabel:`Ordered quantities` as the :guilabel:`Invoicing Policy`.

   .. image:: down_payment/cabinet-product-form-details.png
      :align: center
      :alt: Cabinet with doors product form showcasing various details and fields.

.. seealso::
   :doc:`/applications/sales/sales/invoicing/invoicing_policy`

First, navigate to :menuselection:`Sales app --> New`, and add a :guilabel:`Customer` to the
quotation.

Then, click :guilabel:`Add a product` in the :guilabel:`Order Lines` tab, and select the
:guilabel:`Cabinet with Doors` product.

.. image:: down_payment/cabinet-with-doors-quotation.png
   :align: center
   :alt: Adding cabinet with doors product to a new quotation in the Odoo Sales application.

When the order is confirmed (via the :guilabel:`Confirm` button), the quotation turns into a sales

> order. Then, create/view the invoice, by clicking :guilabel:`Create Invoice`.

Reviewer 2: ```suggestion
order. Once this occurs, create and view the invoice by clicking :guilabel:`Create Invoice`.
```

.. image:: down_payment/cabinet-sales-orders-confirmed.png
   :align: center
   :alt: Cabinet with doors sales order that's been confirmed in the Odoo Sales application.

Next, on the :guilabel:`Create invoices` pop-up window that appears, select :guilabel:`Down payment
(percentage)`, and type `50` in the :guilabel:`Down Payment Amount` field.

.. note::
   The :guilabel:`Income Account` and :guilabel:`Customer Taxes` fields are *not* required fields,

>    and they will *not* appear if they've already preconfigured in previous down payment requests.

Reviewer 2: ```suggestion
   and they will *not* appear if they've already been preconfigured in previous down payment requests.
```

   For more information, check out the documentation on :ref:`customer taxes modification on down
   payments <sales/invoicing/down_payment/customer-tax-modification-down-payments>` and :ref:`income
   account modification on down payments
   <sales/invoicing/down_payment/income-account-modification>`.

Lastly, click :guilabel:`Create Draft Invoice` to create and view the invoice draft.

.. image:: down_payment/create-invoices-percentage-popup.png
   :align: center
   :alt: Create invoices pop-up window with 50% percentage fields filled in with Odoo Sales.

Clicking :guilabel:`Create Draft Invoice` reveals the draft invoice, which includes the down
payment as a :guilabel:`Product` in the :guilabel:`Invoice Lines` tab.

.. image:: down_payment/draft-invoice-sample.png
   :align: center
   :alt: A sample draft invoice with down payment mentioned in Odoo Sales.

> From there, the invoice can be confirmed/posted by clicking :guilabel:`Confirm`. Confirming the

Reviewer 2: ```suggestion
From there, the invoice can be confirmed and posted by clicking :guilabel:`Confirm`. Confirming the
```

> invoice, changes the status from :guilabel:`Draft` to :guilabel:`Posted`. It also reveals a new

Reviewer 2: ```suggestion
the invoice changes the status from :guilabel:`Draft` to :guilabel:`Posted`. It also reveals a new
```

.. image:: down_payment/customer-invoice-green-payment-banner.png
   :align: center
   :alt: Customer Invoice with a green In Payment banner located in the upper-right corner.

Now, when the customer wants to pay the remaining amount of the order, another invoice must be
created. To do that, return to the sales order, via the breadcrumb links.

Back on the sales order, a new :guilabel:`Down Payments` section is present in the :guilabel:`Order
Lines` tab, along with the :guilabel:`Down payment` that was just invoiced and posted.

.. image:: down_payment/down-payments-section-order-lines.png
   :align: center
   :alt: The down payments section in the order lines tab of a sales order.

Next, click the :guilabel:`Create Invoice` button.

On the :guilabel:`Create invoices` pop-up window that appears, there are two new fields:
:guilabel:`Already invoiced` and :guilabel:`Amount to invoice`.

.. image:: down_payment/create-invoices-pop-up-already-invoiced.png
   :align: center
   :alt: The deduct down payment option on the create invoices pop up in Odoo Sales.

If the remaining amount is ready to be paid, select the :guilabel:`Regular Invoice` option. Odoo

> will create an invoice the exact amount needed to complete the total payment, as indicated in the

Reviewer 2: ```suggestion
will create and invoice the exact amount needed to complete the total payment, as indicated in the
```
Right?

> :guilabel:`Amount to invoice` field.

Reviewer 2: ```suggestion
the :guilabel:`Amount to invoice` field.
```

> Then, click :guilabel:`Create Draft Invoice`.

Reviewer 2: ```suggestion
Once ready, click :guilabel:`Create Draft Invoice`.
```
I just think this phrase helps this part to flow easier/make more sense

the necessary information related to each invoice.

.. image:: down_payment/second-down-payment-invoices-page.png
   :align: center
   :alt: The second down payment invoices page in Odoo Sales.

To complete the flow, click :guilabel:`Confirm`, which changes the status of the invoice from
:guilabel:`Draft` to :guilabel:`Posted`. Then, click :guilabel:`Register Payment`.

Once again, the :guilabel:`Register Payment` appears, with all fields auto-populated with the
necessary information, including the remaining amount left to be paid on the order.

.. image:: down_payment/second-register-payment-popup.png
   :align: center
   :alt: The second register payment pop-up form in Odoo sales.

After confirming that information, click :guilabel:`Create Payment`. Doing so reveals the final
:guilabel:`Customer Invoice` with a green :guilabel:`In Payment` banner in the upper-right corner.
Also, both down payments are present in the :guilabel:`Invoice Lines` tab.

.. image:: down_payment/second-down-payment-in-payment-invoice.png
   :align: center
   :alt: The second down payment invoice with in payment banner in Odoo Sales.

> And the flow is now complete.

Reviewer 2: I think we could do without this sentence personally, but feel free to keep it if you think it helps to separate from the blocks below

Author: I reworded it slightly just to clearly indicate to the reader that the flow they just read through is now complete.

remaining amount at a later date. In turn, the seller provides goods or services to the buyer after
accepting the down payment, trusting that the remaining amount will be paid later on.

Create invoices
===============

When a sales order is confirmed, the option to create an invoice becomes available, via the
:guilabel:`Create Invoice` button, located in the upper-left corner of the sales order form. When
clicked, a :guilabel:`Create invoices` pop-up appears.

.. image:: down_payment/create-invoices-popup-form.png
   :align: center
   :alt: Create invoices pop-up form that appears in Odoo Sales.

.. note::
   Invoices are automatically created as drafts, so they can be reviewed before validation.

On the :guilabel:`Create invoices` pop-up, there are 3 options to choose from in the
:guilabel:`Create Invoice` field:

- :guilabel:`Regular invoice`
- :guilabel:`Down payment (percentage)`
- :guilabel:`Down payment (fixed amount)`

.. note::
   If :guilabel:`Regular Invoice` is selected, the other fields disappear, as they only pertain to
   down payment configurations.

>    .. image:: down_payment/create-invoices-popup-form-regular-option.png
>       :align: center
>       :alt: The Regular Invoice option selected on the Create invoices pop-up form.

Reviewer 3: To cut down on the number of images in this doc, I think that this one can be omitted
```suggestion
```

.. image:: down_payment/down-payment-product-invoice-draft.png
   :align: center
   :alt: Down payment as a product in the invoice lines tab of a customer invoice draft in Odoo.

.. note::
   When the :guilabel:`Down payment` product in the :guilabel:`Invoice Lines` tab is clicked, Odoo
   reveals the product form for the down payment.

   By default, the :guilabel:`Product Type` of down payment products generated for invoices are set
   as :guilabel:`Service`, with the :guilabel:`Invoicing Policy` set to :guilabel:`Prepaid/Fixed
   Price`.

   .. image:: down_payment/down-payment-product-form.png
      :align: center
      :alt: Down payment product form with service product type and invoicing policy field.

   This product can be edited/modified at any time.

.. warning::
   If :guilabel:`Based on Delivered Quantity (Manual)` is chosen as the :guilabel:`Invoicing
   Policy`, an invoice will **not** be able to be created.

Example: request 50% down payment
=================================

.. note::
   The following example involves a 50% amount down payment on a product (:guilabel:`Cabinet with
   Doors`) with :guilabel:`Ordered quantities` as the :guilabel:`Invoicing Policy`.

>    .. image:: down_payment/cabinet-product-form-details.png

Reviewer 3: Image can be cropped

![down-payment-product-form](https://github.com/odoo/documentation/assets/36018073/13cc7672-0465-47fd-9046-cdd15b577992)

      :align: center
      :alt: Cabinet with doors product form showcasing various details and fields.

.. seealso::

>    :doc:`/applications/sales/sales/invoicing/invoicing_policy`

Reviewer 3: Update doc path to be relative
```suggestion
   :doc:`invoicing_policy`
```

First, navigate to :menuselection:`Sales app --> New`, and add a :guilabel:`Customer` to the
quotation.

Then, click :guilabel:`Add a product` in the :guilabel:`Order Lines` tab, and select the
:guilabel:`Cabinet with Doors` product.

> .. image:: down_payment/cabinet-with-doors-quotation.png

Reviewer 3: optional - remove this image too to cut down on images
```suggestion
```

   :align: center
   :alt: Adding cabinet with doors product to a new quotation in the Odoo Sales application.

When the order is confirmed (via the :guilabel:`Confirm` button), the quotation turns into a sales
order. Once this occurs, create and view the invoice by clicking :guilabel:`Create Invoice`.

.. image:: down_payment/cabinet-sales-orders-confirmed.png
   :align: center
   :alt: Cabinet with doors sales order that's been confirmed in the Odoo Sales application.

Next, on the :guilabel:`Create invoices` pop-up window that appears, select :guilabel:`Down payment
(percentage)`, and type `50` in the :guilabel:`Down Payment Amount` field.

.. note::
   The :guilabel:`Income Account` and :guilabel:`Customer Taxes` fields are *not* required fields,
   and they will *not* appear if they've already been preconfigured in previous down payment
   requests.

   For more information, check out the documentation on :ref:`customer taxes modification on down

>    payments <sales/invoicing/down_payment/customer-tax-modification-down-payments>` and :ref:`income
>    account modification on down payments
>    <sales/invoicing/down_payment/income-account-modification>`.

Reviewer 3: No need to include filename in the ref paths
```suggestion
   payments <sales/invoicing/customer-tax-modification-down-payments>` and :ref:`income account
   modification on down payments <sales/invoicing/income-account-modification>`.
```
